### PR TITLE
Fix uml permissions

### PR DESCRIPTION
**Pull request by wd60622**
## Related Issue

- [ ] Closes #
- [ ] Related to #

## Checklist

- [ ] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [ ] Added necessary documentation (docstrings and/or example notebooks)
- [ ] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Modules affected

- [ ] MMM
- [ ] CLV

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [ ] Documentation
- [ ] Maintenance
- [ ] Other (please specify):

----
📚 Documentation preview 📚: https://pymc-marketing--913.org.readthedocs.build/en/913/

### CODE REVIEW

File: .github/workflows/uml.yml [hunk 1]
```diff
@@ -10,12 +10,10 @@ on:
...
-permissions: write-all
-
 jobs:
   build:
     runs-on: ubuntu-latest
-
+    permissions: write-all
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
```
Comment (wd60622) [inline .github/workflows/uml.yml:19]: Will try v4 instead

Also, make sure that the checked out hash is good to push and doesnt require additional pull

**Review by juanitorduz — APPROVED** [review]
Let’s try it out on main and if fails we revert ok @wd60622 😄?

**Comment by wd60622** [comment]
Think we have a winner @juanitorduz 🎉

**Comment by juanitorduz** [comment]
Yay! Thanks for going into this rabbit whole 😅

**Pull request merged**